docs/configuration/index.md

```diff
@@ -1358,8 +1358,8 @@ You can optionally configure following additional configs to restrict druid inge
  
 |Property|Possible Values|Description|Default|
 |--------|---------------|-----------|-------|
-|`druid.ingestion.http.allowListDomains`|List of domains|Allowed domains from which ingestion will be allowed. Only one of allowList or denyList can be set.|empty list|
```

Comment:
yes, they were never released and just committed to master. 
I removed them since having a generic configs that apply would to all inputSources would be easier to configure instead of having source specific configs. .